hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -1478,6 +1478,17 @@
     <value>-Xmx256m</value>
   </property>
 
+  <property>
+    <name>yarn.nodemanager.container-localizer.java.opts.add-exports-as-default</name>
+    <value>true</value>
+    <description>Since on JDK17 it's no longer possible to use the reflection API to
+      access public fields and methods add-exports flags should be added to container
```

Comment:
Nit: comma should be added after: "field and methods"

Reply:
Fixed